modules/data_release/ajax/GetFile.php

```diff
@@ -58,8 +58,12 @@
     header("HTTP/1.1 403 Forbidden");
     exit;
 }
-$fp = fopen($FullPath, 'r');
-fpassthru($fp);
-fclose($fp);
+
+// Output file in downloadable format
+header('Content-Description: File Transfer');
+header('Content-Type: application/force-download');
```

Comment:
See the PRs linked in the description. They also contain `application/force-download`.